website/docs/llm_configuration.ipynb

```diff
@@ -0,0 +1,350 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "llm_config = (\n",
+    "    {\n",
+    "        \"config_list\": [{\"model\": \"gpt-4\", \"api_key\": os.environ[\"OPENAI_API_KEY\"]}],\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "````{=mdx}\n",
+    ":::warning\n",
+    "It is important to never commit secrets into your code, therefore we read the OpenAI API key from an environment variable.\n",
+    ":::\n",
+    "````\n",
+    "\n",
+    "This `llm_config` can then be passed to an agent's constructor to enable it to use the LLM."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import autogen  # noqa: E402\n",
+    "\n",
+    "assistant = autogen.AssistantAgent(name=\"assistant\", llm_config=llm_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## `config_list`\n",
+    "\n",
+    "Different tasks may require different models, and the `config_list` allows specifying the different endpoints and configurations that are to be used. It is a list of dictionaries, each of which contains the following keys depending on the kind of endpoint being used:\n",
+    "\n",
+    "````{=mdx}\n",
+    "import Tabs from '@theme/Tabs';\n",
+    "import TabItem from '@theme/TabItem';\n",
+    "\n",
+    "<Tabs>\n",
+    "  <TabItem value=\"openai\" label=\"OpenAI\" default>\n",
+    "    - `model` (str, required): The identifier of the model to be used, such as 'gpt-4', 'gpt-3.5-turbo'\n",
+    "    - `api_key` (str, required): The API key required for authenticating requests to the model's API endpoint.\n",
+    "    - `api_type`: `openai`. This is optional and defaults to `openai`.\n",
+    "    - `base_url` (str, optional): The base URL of the API endpoint. This is the root address where API calls are directed\n",
+    "    - `tags` (List[str], optional): Tags which can be used for filtering\n",
+    "\n",
+    "    Example:\n",
+    "    ```json\n",
+    "    [\n",
+    "      {\n",
+    "        \"model\": \"gpt-4\",\n",
+    "        \"api_key\": os.environ['OPENAI_API_KEY']\n",
+    "      }\n",
+    "    ]\n",
+    "    ```\n",
+    "  </TabItem>\n",
+    "  <TabItem value=\"azureopenai\" label=\"Azure OpenAI\">\n",
+    "    - `model` (str, required): The identifier of the model to be used, such as 'gpt-4', 'gpt-3.5-turbo'\n",
+    "    - `api_key` (str, required): The API key required for authenticating requests to the model's API endpoint\n",
+    "    - `api_type`: `azure`\n",
+    "    - `base_url` (str, optional): The base URL of the API endpoint. This is the root address where API calls are directed\n",
+    "    - `api_version` (str, optional): The version of the Azure API you wish to use\n",
+    "    - `tags` (List[str], optional): Tags which can be used for filtering\n",
```

Comment:
missing period

Reply:
I usually opt to leave out periods for non-complete sentences for bullet points,  but i broke that rule for the `base_url` line

Reply:
If you'd rather consistently have periods on each line I can make that change though

Reply:
I just want consistency. It's not the case that the example below is specific to tags. So that sentence is complete.

Reply:
Updated the bullet points to add periods